### DISCUSSION TITLE
Game Crashes on startup

### DISCUSSION THREAD

**Discussion by SpikerzZ406** [julianxhokaxhiu/FFNx#540] [Q&A] [answered]
it was working yesterday no problem, played today and now I get an error when starting the game

"Feel free to visit this link to know about further next steps you can take
https://github.com/julianxhokaxhiu/FFNx/blob/master/docs/faq.md"

All I did was update from 1.15.0.1 to 1.15.0.12, I then tried to revert the update back and still not working
I attached the log file

[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/10687027/FFNx.log)

**Comment by eve-atum** [comment, marked as answer]
Sounds like you may have had a Windows patch cause a reboot overnight, and in true MSFT fashion, they've changed security settings without your knowledge or consent.  If you're on either the Qhimm or Tsunamods discord, issue the "/dep" command in one of the FF7 support channels to see how to check (and possibly revert) that setting.

**Comment by SpikerzZ406** [comment]
Thanks for the help, while I was waiting for a response I uninstalled FF VII and deleted the folder, then reinstalled it with FFNX version 1.15.0.0 and now it is working again

**Comment by leorob88** [comment]
I found an issue with crash in a strange way. While I started using properly the mods I think only after Midgar, I still played smoothly until the end of disc 2. BUT when going to see Zack's flashback, the game crashes when trying to load the map where Zack dies.
I know now that specifically the mod ChaOS (replace characters models with more detailed and realistic ones) is causing this for me, i'm trying now to do some tests configuring the mod with different options to see if there's a specific option causing the crash or it's just the mod. I'd like to find out also because this mod is followed by 2 other mods, so I'd like to make it work. I'll report if I can make it work or not.

**Reply by leorob88** [reply]
So, I made some tests and the culprit seems indeed the option to scale up the models to fit better the surroundings. Also, I have the feeling the "sword model" option is not consistent because it doesn't work as it states (the options are there, only with wrong names). For this flashback to work correctly I had to set the sword as "preparing for battle" to remove the sword from Cloud's back in that scene, as it uses the model Cloud has when he's in towns etc.
Strange enough, I noticed that Normal and Always seem to give Cloud always the sword regardless, while Preparing for battle hides the sword, it seems, only when in towns and such places where you're not supposed to fight monsters, I suppose. But inside Shinra manor instead the sword is back. I suppose it makes sense somehow with "Preparing for battle". But this is more a detail for the mod settings rather than a way to avoid crash.
